lib/shared/addon/components/cluster-driver/driver-rke/component.js

```diff
@@ -332,6 +383,24 @@ export default InputTextFile.extend(ClusterDriver, {
       out += ` --label ${ key }=${ labels[key] }`;
     });
 
+    if (!get(this, 'isLinux')) {
```

Comment:
`get(this, 'isWindows')`

Reply:
also this doesn't seem to be working because the command doesn't show up in the ui